.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "ngrx-traits",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+		  "VARIANT": "20-bullseye",
+		  "EXTRA_NODE_VERSION": "lts/iron"
+		}
+	  },
+	"features": {
+		"ghcr.io/devcontainers-contrib/features/nx-npm:1": {}
+	},
+	"postCreateCommand": "sh .devcontainer/hooks/postCreateCommand.sh",
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"angular.ng-template",
+				"nrwl.angular-console",
+				"esbenp.prettier-vscode",
+				"firsttris.vscode-jest-runner",
+				"dbaeumer.vscode-eslint"
+			]
+		}
+	},
+	"mounts": [
+		"source=ngrx-traits-angular-cache,target=${containerWorkspaceFolder}/.angular,type=volume", // angular cache volume
+		"source=ngrx-traits-nx-cache,target=${containerWorkspaceFolder}/.nx,type=volume", // angular cache volume
+		"source=ngrx-traits-tmp,target=${containerWorkspaceFolder}/tmp,type=volume", // tmp volume
```

Comment:
What is the tmp folder inside the project for ? ${containerWorkspaceFolder}/tmp

Reply:
To be honest? I don't remember. I'm going to remove it